packages/devtools_app/lib/main.dart

```diff
@@ -40,3 +40,37 @@ void main() {
     const DevToolsApp(screens),
   );
 }
+
+// Example of conditional screen that could be added to [screens] above.
```

Comment:
Code examples should use the dartdoc style for including code snippets used by Flutter.